### DISCUSSION TITLE
Save as PNG

### DISCUSSION THREAD

**Discussion by PeterPedro** [mifi/lossless-cut#1322] [Ideas]
Just a feature request, to save as PNG frames just to make the process of extracting lossless frames easier

**Comment by mifi** [comment]
Isn't this already supported? or can you explain a bit more detailed where you want PNG support?

**Reply by PeterPedro** [reply]
PNG is supported either as saving single files or APNG (not working for me btw) but I meant saving as multiple PNG files, just to make the process easier because there could be lots of frames and saving them one by one is tedious. I use it for extracting frames, just a suggestion.

**Comment by mifi** [comment]
you mean the "extract all frame as images" function? that function will respect the image format setting, similar to capture single frame.

**Reply by PeterPedro** [reply]
Oh I never noticed that function, thank you!
